scripts/download_wikidata_drainagebasins.py

```diff
@@ -0,0 +1,61 @@
+import requests
+from tqdm import tqdm
+import json
+import re
+
+url = 'https://query.wikidata.org/sparql'
+subquery = """
+SELECT ?item ?itemLabel ?altLabel
+WHERE 
+{
+  ?item wdt:P31 wd:Q166620.
+  ?article schema:about ?item.
+  OPTIONAL { ?item skos:altLabel ?altLabel . FILTER (lang(?altLabel) = "en") }
+  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
+  FILTER NOT EXISTS {?item wdt:P31 wd:Q23397.}
+}
+GROUP BY ?item ?itemLabel ?altLabel
+"""
+#combinations = [f"Q{n}" for n in range(10, 100)]
+combinations = ['everything']
+waterbodies = {}
+for comb in tqdm(combinations):
+    #query = subquery + f"'{comb}'))" + "} GROUP BY ?item ?itemLabel ?altLabel"
+    query = subquery
+    try:
+        r = requests.get(url, params = {'format': 'json', 'query': query})
+        data = r.json()
+        for entry in data["results"]["bindings"]:
+            key = entry["item"]["value"].split("/")[-1]
+            label = entry['itemLabel']['value'] if 'itemLabel' in entry else ""
+            alias = entry['altLabel']['value'] if 'altLabel' in entry else ""
+            
+            if not (re.search('^Q[0-9]+', label) or label == ""):
+                if key not in waterbodies:
+                    waterbodies[key] = set()
+                waterbodies[key].add(label.lower())
+            
+            if not (re.search('^Q[0-9]+', alias) or alias == ""):
+                if key not in waterbodies:
+                    waterbodies[key] = set()
+                waterbodies[key].add(alias.lower())
+    except Exception as e:
```

Comment:
it's generally preferred to not catch broadly encompassing `Exception`. can we add a `TODO: make this more specific` please

Reply:
Added TODO.